roles/letsencrypt/defaults/main.yml

```diff
@@ -23,3 +22,7 @@ intermediate_path: "{{ letsencrypt_cert_dir }}/{{ domain.certificate_name }}_int
 fullchain_path: "{{ letsencrypt_cert_dir }}/{{ domain.certificate_name }}_fullchain.pem"
 private_key_path: "{{ letsencrypt_cert_dir }}/{{ domain.certificate_name }}.key"
 remaining_days: "30"
+
+### dns os (openstack/otc)
+dns_os_auth_url: "https://iam.eu-de.otc.t-systems.com:443/v3"
```

Comment:
I'm not sure we should set a default here.

Reply:
253bb01